client/src/Views/Home/Navbar/HomeNavbar.js

```diff
@@ -12,26 +13,26 @@ const styles = () => ({
     padding: "0 48px",
     height: 100
   },
+  logo: {
+    flexGrow: 1
+  },
   tabs: {
-    height: "100%"
+    height: "100%",
+    paddingRight: "10rem"
   }
 });
 
 const HomeNavbar = props => (
-  <Grid
-    item
-    container
-    alignItems="center"
-    justify="space-between"
-    className={props.classes.root}
-  >
-    <Grid item>
+  <Grid item container alignItems="center" className={props.classes.root}>
+    <Grid item className={props.classes.logo}>
```

Comment:
we should use the hooks API for styling as opposed to the withStyles HOC - https://material-ui.com/styles/basics/#hook-api, when we are using a functional component

Reply:
Change this throughout this PR

Reply:
In the interest of having things ready for the presentation, I'll make a separate issue for this to work on later